jedi/_compatibility.py

```diff
@@ -18,6 +19,19 @@
 is_py26 = not is_py3 and sys.version_info[1] < 7
 
 
+class DummyFile(object):
```

Comment:
New instance of `DummyFile` is getting created on each import lookup, even if have done it before. Is it a problem?